rtrlib/spki/hashtable/ht-spkitable.c

```diff
@@ -288,3 +288,89 @@ int spki_table_src_remove(struct spki_table *spki_table,
     pthread_rwlock_unlock(&spki_table->lock);
     return SPKI_SUCCESS;
 }
+
+int spki_table_copy_except_socket(struct spki_table *src, struct spki_table *dst, struct rtr_socket *socket)
+{
+    tommy_node *current_node;
+    int ret = SPKI_SUCCESS;
+
+    pthread_rwlock_rdlock(&src->lock);
+    current_node = tommy_list_head(&src->list);
+    while (current_node) {
+        struct key_entry *entry;
+        struct spki_record record;
+
+        entry = (struct key_entry *) current_node->data;
+        key_entry_to_spki_record(entry, &record);
+
+        if (entry->socket == socket) {
```

Comment:
Shouldn't this be entry->socket != socket ?

Reply:
Yes, it should …

Reply:
how about fix it 😄

Reply:
whuups I thought I already did …